### DISCUSSION TITLE
Not seeing the newer features of 2.1.1?

### DISCUSSION THREAD

**Discussion by youradds** [plausible/analytics#4614] [Self-Hosted Support]
Hi,

As far as I'm aware, I have updated to 2.1.1, yet I don't see the new options:

https://plausible.io/docs/excluding

All I get is:

![image](https://github.com/plausible/analytics/assets/2000191/fcfde749-2672-4c05-bab7-f42b8e83ed51)

Am I missing something? This is self-hosted. I've done:

```
docker-compose stop plausible plausible_events_db
docker-compose rm plausible plausible_events_db
docker-compose up
```

On another notes - the last few days I'm seeing a HUGE amount of traffic coming in. Its all/most coming from 1 seconds sessions - and in "Direct / None" the category - so I'm not sure where its coming from. I'm wondering if there is someone going through and spamming the site? 

![image](https://github.com/plausible/analytics/assets/2000191/54b909b8-887e-4cb2-9b26-ba20b1b58555)

Thanks!

Andy

**Comment by ruslandoga** [comment]
👋 @youradds 

This doesn't seem to be an issue with the documentation. I'm transferring it to our discussions.

**Comment by ruslandoga** [comment]
Shields should be present in v2.1.1: https://github.com/plausible/analytics/blob/v2.1.1/lib/plausible_web/views/layout_view.ex#L63-L74

It looks like your screenshot might be from an earlier version, like v2.0.0: https://github.com/plausible/analytics/blob/v2.0.0/lib/plausible_web/views/layout_view.ex#L20-L39
